test/unit/test_resubmit.js

```diff
@@ -37,12 +37,12 @@ describe("ResubmitController", function () {
         });
 
         beforeEach(inject(function($injector) {
-            
+
             $timeout = $injector.get('$timeout');
             $httpBackend = $injector.get('$httpBackend');
-            $httpBackend.whenPOST("/api/resubmit/1").respond(
+            $httpBackend.whenPOST("/api/testjobs/1/resubmit/").respond(
                 200, ["submitted"]);
-            $httpBackend.whenPOST("/api/forceresubmit/1").respond(
+            $httpBackend.whenPOST("/api/testjobs/1/force_resubmit/").respond(
```

Comment:
Same here

Reply:
Hmm, this is only change of the URL. The rest comes from original test. This line was changed in df8482eb and the next one in f2e9d53e. @terceiro, @stevanradakovic could you explain why the test checks 401 response?